.github/workflows/main.yaml

```diff
@@ -0,0 +1,34 @@
+name: Test yugabyte-db-action
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  test-yugabytedb-action:
+    runs-on: ubuntu-latest
```

Comment:
would be good to also run it on centos:latest

Reply:
Github only supports macOS, windows, and ubuntu as an OS for Github actions. 
Reference Doc: https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners